packages/styles/combobox.css

```diff
@@ -0,0 +1,246 @@
+    0 0 5px var(--combobox-input-border-focus-glow-color),
+    inset 0 1px 2px rgba(0, 0, 0, 0.05);
+}
+.Combobox__arrow {
+  width: 2rem;
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  right: 0;
+  pointer-events: none;
+}
+
+.Combobox__arrow:before {
+  content: '';
+  display: inline-block;
+  border-top: 7px solid currentColor;
+  border-left: 6px solid transparent;
+  border-right: 6px solid transparent;
+  border-radius: 2px;
+}
+
+.Combobox__listbox {
+  display: none;
+  position: absolute;
+  min-width: 100%;
+  border: 1px solid var(--combobox-listbox-border-color);
+  box-shadow: 0 2px 3px 0 rgba(0, 0, 0, 0.15);
+  background-color: var(--combobox-listbox-background-color);
+  z-index: var(--z-index-listbox);
+  max-width: var(--combobox-listbox-max-width, 100%);
+  max-height: var(
+    --combobox-listbox-max-height,
+    calc(var(--combobox-option-min-height) * 10)
+  );
+  overflow-y: auto;
+  animation: combobox-animate-in ease-out 150ms;
+}
+
+@keyframes combobox-animate-in {
+  from {
+    transform: translateY(-8px);
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@media (prefers-reduced-motion) {
+  .Combobox__listbox {
+    animation: none;
+  }
+}
+
+.Combobox__listbox:hover {
+  border-color: var(--combobox-listbox-border-color);
+}
+
+/* 
+ * Note: Firefox doesn't yet have full support for :has, so .Combobox__listbox--open
+ * is provided as a fallback selector to indicate open listboxes
+ * see: https://caniuse.com/css-has
+ */
+.Combobox:has([role='combobox'][aria-expanded='true']) .Combobox__listbox,
+.Combobox__listbox--open {
+  display: block;
+}
+
+.Combobox__listbox li {
+  list-style-type: none;
+}
+
+.Combobox__listbox > ul[role='group']:not(:last-child) {
+  border-bottom: 1px solid var(--combobox-listbox-border-color);
+}
+
+.ComboboxGroup__label {
+  background-color: var(--combobox-group-label-background-color);
+  padding: var(--space-smallest);
+  font-size: var(--combobox-group-label-font-size);
+  font-weight: var(--font-weight-medium);
+  border-bottom: 1px solid var(--combobox-listbox-border-color);
+}
+
+.ComboboxGroup--hidden {
+  display: none !important;
+}
+
+.ComboboxOption {
+  color: var(--combobox-option-font-color);
+  padding: var(--space-smallest);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  gap: var(--space-half);
+  justify-content: space-between;
+  min-height: var(--combobox-option-min-height);
+}
+
+.ComboboxOption:not(:last-child) {
+  border-bottom: 1px solid var(--combobox-listbox-border-color);
+}
+
+.ComboboxOption--active {
+  box-shadow: inset 0 0 0 1px var(--combobox-listbox-background-color),
+    inset 0 0 0 3px var(--focus);
+}
+
+.ComboboxOption:is(:hover, [role='option'][aria-selected='true']:hover) {
+  background-color: var(--combobox-option-hover-background-color);
+}
+
+.ComboboxOption .Icon {
+  color: var(--combobox-option-selected-icon-color);
+}
+
+.ComboboxOption .Icon svg {
+  padding: 4px;
```

Comment:
```suggestion
  padding: var(--space-half);
```

Reply:
This should _not_ use the spacing vars here since this padding is a fixed padding relative to the icon size and not the design system spacing.